zazu/repo/commands.py

```diff
@@ -67,12 +69,17 @@ def clone(ctx, repository_url, nohooks):
         --nohooks: does not install git hooks in the cloned repo
     """
     try:
-        git.Git().clone(repository_url)
+        repo = git.Repo.clone_from(repository_url, '{}/{}'.format(os.getcwd(), repository_url.rsplit('/', 1)[-1].replace('.git', '')))
```

Comment:
So, it appears i was using git python wrong before. The previous way returns no usable objects, it just clones the repo. According to the documentation this is the proper way to use it, and allows a quick and easy way to access other git functionality from within the repo object.